application/core/database/matching-stats.go

```diff
@@ -0,0 +1,96 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/koki/randommatch/entity"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j/dbtype"
+)
+
+func CreateMatchingStat(MatchingStats entity.MatchingStat) (string, error) {
+	driver, err := Driver()
+	if err != nil {
+		return "", err
+	}
+	session := (*driver).NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	uid, err := session.WriteTransaction(func(tx neo4j.Transaction) (interface{}, error) {
+		result, err := tx.Run("MERGE (n:MatchingStat {uid: $id}) "+
+			"ON CREATE SET n += {uid: $id, num_groups: $numgroups, num_conversations: $numconvs, num_failures: $numfailed, "+
+			"creation_date: $created_at, last_update: datetime({timezone: 'Z'})} "+
+			"RETURN n.uid",
+			map[string]interface{}{
+				"id":         uuid.New().String(),
+				"numgroups":  MatchingStats.NumGroups,
+				"numconvs":   MatchingStats.NumConversations,
+				"numfailed":  MatchingStats.NumFailed,
+				"created_at": time.Now().UTC(),
```

Comment:
do not do this put directly neo4j datetime as you did for last_update above

Reply:
I do this because I created a new field called CreatedAt on the MatchingStat entity to be able to know the date of a matchingStat. I check this and it gives the same result as datetime

Reply:
you need to be consistent and better use datetime in the query as it is done for last_update